libafl/src/state/mod.rs

```diff
@@ -703,6 +708,7 @@ where
         manager: &mut EM,
         forced: bool,
         loader: &mut dyn FnMut(&mut Z, &mut Self, &Path) -> Result<I, Error>,
+        exit_on_solution: bool,
```

Comment:
Can we either add a `LoadConfig` struct with the bools and loader in it (probably pref this one), or move the `loader` up so the booleans are next to each other?